src/main/java/com/hartwig/platinum/storage/OutputBucket.java

```diff
@@ -37,6 +37,7 @@ public String findOrCreate(final String runName, final String region, final Plat
                     .setIamConfiguration(BucketInfo.IamConfiguration.newBuilder().setIsUniformBucketLevelAccessEnabled(true).build())
                     .setLocation(region)
                     .setDefaultKmsKeyName(configuration.cmek().orElse(null))
+                    .setAutoclass(BucketInfo.Autoclass.newBuilder().setEnabled(true).build())
```

Comment:
The default terminal storage class is `Nearline`. Would it make sense to use the even cheaper (but more expensive to retrieve) `Archive` as the final storage class?